client/settings/settings-manager/index.js

```diff
@@ -92,6 +93,7 @@ const TransactionsAndDepositsDescription = () => (
 const SettingsManager = () => {
 	const {
 		featureFlags: { upeSettingsPreview: isUPESettingsPreviewEnabled },
+		featureFlags: { upe: isUpeEnabled },
```

Comment:
(minor) Something I didn't realize before - no need to deconstruct `featureFlags` again, since it's already deconstructed

```suggestion
		featureFlags: { upeSettingsPreview: isUPESettingsPreviewEnabled, upe: isUpeEnabled, },
```

Reply:
Good point - nested de-structuring still gets me 😖